sgv2-service-common/src/main/java/io/stargate/sgv2/common/grpc/DefaultStargateBridgeClient.java

```diff
@@ -107,6 +110,91 @@ public CompletionStage<Response> executeQueryAsync(Query query) {
     return observer.asFuture();
   }
 
+  @Override
+  public CompletionStage<Response> executeQueryAsync(
+      String keyspaceName,
+      String tableName,
+      Function<Optional<Schema.CqlTable>, Query> queryProducer) {
+    String decoratedKeyspaceName = addTenantPrefix(keyspaceName);
+    CqlKeyspaceDescribe keyspace = keyspaceCache.getIfPresent(decoratedKeyspaceName);
+    if (keyspace == null) {
+      // Nothing in our local cache, check if the bridge has a more recent version
+      return forceFetchAndExecute(keyspaceName, decoratedKeyspaceName, tableName, queryProducer);
+    } else {
+      // Build the query with our local version
+      Query query;
+      try {
+        query = produceQuery(queryProducer, Optional.of(keyspace), tableName);
+      } catch (Throwable t) {
```

Comment:
Is there a reason this is not `Exception` but `Throwable`? (I feel I may have asked same question before,
and it had to do with custom subtypes or something?)

Reply:
Yeah, it's unlikely that retrying on something like `Error` would be useful. Changing it.